_posts/en/newsletters/2021-03-10-newsletter.md

```diff
@@ -119,14 +119,61 @@ existing software building on top of taproot.
 meeting, highlighting some of the important questions and answers.  Click on a
 question below to see a summary of the answer from the meeting.*
 
-FIXME:jonatack or jnewbery
+[Erlay: bandwidth-efficient transaction relay protocol][review club
+#18261] is a PR ([#18261][Bitcoin Core #18261]) by Gleb Naumenko that proposes
+to implement [BIP330][] in Bitcoin Core.
 
-{% include functions/details-list.md
-  q0="FIXME"
-
-  a0="FIXME"
+The review club discussion focused on the tradeoffs, implementation and
+potential new attack vectors involved with [Erlay][topic erlay]. In subsequent
+meetings, the review club discussed [Minisketch][topic minisketch], a
+[library][minisketch] implementing the PinSketch *set reconciliation* algorithm
+that is the basis for the efficient relay protocol in Erlay.
 
-  a0link="http://example.com/FIXME"
+{% include functions/details-list.md
+  q0="What is Erlay?"
+  a0="A new method of transaction relay based on a combination of *flooding* and
+     *set reconciliation* (the current transaction relay is flooding-only) to
+     improve bandwidth efficiency, scalability, transaction privacy and network
+     security. The idea was presented in a 2019 paper, *[Bandwidth-Efficient
+     Transaction Relay for Bitcoin][erlay paper]*, and is specified in
+     [BIP330][]."
+
+  q1="What advantages does Erlay bring?"
+  a1="[Lower bandwidth use for transaction relay][erlay 1], which comprises
+     roughly half the bandwidth needed to operate a node; [scalability of peer
+     connections][erlay 2], making the network more robust to partitioning
+     attacks and [single nodes more resistant to Eclipse attacks][erlay 3];
+     finally, improved privacy by no longer flooding transactions from private
+     nodes."
```

Comment:
@naumenko I'm not 100% sure about this line about privacy, accuracy check please if you have some spare (txn) bandwidth

Reply:
I think I'd leave this out. I think Erlay may help privacy marginally, but it's hard to quantify and is not the primary motivation.

Reply:
Thanks, that's better--done.